text/pdf_text_color.go

```diff
@@ -0,0 +1,121 @@
+/*
+ * Using RGB and CMYK color to colorize text.
+ *
+ * Run as: go run pdf_text_color.go
+ */
+
+package main
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/creator"
+)
+
+const licenseKey = `
+-----BEGIN UNIDOC LICENSE KEY-----
+Free trial license keys are available at: https://unidoc.io/
+-----END UNIDOC LICENSE KEY-----
+`
+
+func init() {
+	// Enable debug-level logging.
+	// unicommon.SetLogger(unicommon.NewConsoleLogger(unicommon.LogLevelDebug))
+
+	err := license.SetLicenseKey(licenseKey, `Company Name`)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func main() {
+	c := creator.New()
+	c.SetPageMargins(50, 50, 100, 70)
+
+	colorRGB(c)
+	colorCMYK(c)
+
+	if err := c.WriteToFile("color.pdf"); err != nil {
+		fmt.Printf("Error %v", err)
+	}
+}
+
+func drawTextRGB(c *creator.Creator, chapter *creator.Chapter,
```

Comment:
can we reduce this to like 2 functions... in addition to main.  Maybe add a simple shape also with color.
Can also put outline on the text or something with color also to illustrate